extpsdk/src/main/python/vantiqconnectorsdk.py

```diff
@@ -220,7 +223,15 @@ def __init__(self, source_name: string, config: Union[dict, None]):
             if kwArgString is not None:
                 try:
                     kw_temp : dict = json.loads(kwArgString)
-                    self.connect_kw_args = kw_temp
```

Comment:
when we get this special case, construct an SSL context to be used for the connection.  

`ssl=false` disables ssl (so our server refuses to connect), and I don't see a way to do this in pure JSON.  So we special case for this a bit.